src/views/common/table/ObjectTable.tsx

```diff
@@ -116,6 +118,17 @@ function ObjectTable<T extends ObjectData>({ objects, columns }: Props<T>) {
           ))}
         </tbody>
       </table>
+
+      {/* If there is exactly one object after filtering and sorting, display its full details
+          below the table. This gives users more information when only one result
+          remains. We don't use the sliceFunction here because we want to base
+          the condition on all filtered objects, not just the ones shown on the
+          current page. */}
+      {objectsFilteredAndSorted.length === 1 && (
+        <div style={{ marginTop: '1em' }}>
+          <ObjectDetails object={objectsFilteredAndSorted[0]} />
```

Comment:
Looks great! Let's add the full details container though so it 1) uses horizontal margins and 2) shows the object title.

```suggestion
          <DetailsContainer title={<ObjectTitle object={objectsFilteredAndSorted[0]} />}>
            <ObjectDetails object={objectsFilteredAndSorted[0]} />
          </DetailsContainer>
```